src/yggdrasil/wire.go

```diff
@@ -42,15 +42,6 @@ func wire_put_uint64(e uint64, out []byte) []byte {
 	return append(out, b[i:]...)
 }
 
-// Returns the length of a wire encoded uint64 of this value.
-func wire_uint64_len(elem uint64) int {
-	l := 1
-	for e := elem >> 7; e > 0; e >>= 7 {
-		l++
-	}
-	return l
-}
```

Comment:
I think we should keep this. We happen to not use it at the moment, but it will almost certainly be needed in the `future` when we source route.